backend-lib/build.gradle.kts

```diff
@@ -84,6 +84,9 @@ cargo {
     )
     profile = "release"
     prebuiltToolchains = true
+    exec = { spec, _ ->
+        spec.environment["RUST_ANDROID_GRADLE_CC_LINK_ARG"] = "-Wl,-z,max-page-size=16384"
```

Comment:
I would expect this to break compilation, as it is overwriting the environment variable from the cargo build task to not include `-soname`. Does it not do so?

Reply:
It's working for me after this change. Tested on emulators Android SDK API levels 35 and 27 by creating a new wallet, syncing, and retrieving addresses or balances.

Reply:
Well if it works then great.